File: data/submissions/P003/AvgAC.java
import java.util.Locale;
import java.util.Scanner;

/** In 6 chu so thap phan -> nam trong sai so 1e-6 -> AC. */
public class AvgAC {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        double sum = 0;
        for (int i = 0; i < n; i++) sum += sc.nextLong();
        System.out.printf(Locale.US, "%.6f%n", sum / n);
    }
}

File: data/submissions/P003/avg_ac.py
# In tan 9 chu so thap phan - vi du 2.333333333 so voi dap an 2.333333.
# Sai so 3.3e-10 < 1e-6 nen van AC: minh hoa FloatComparator.
import sys

data = sys.stdin.read().split()
n = int(data[0])
nums = [int(x) for x in data[1:1 + n]]
print("%.9f" % (sum(nums) / n))
